app/styles/reset.styl

```diff
@@ -0,0 +1,78 @@
+// Based on [Eric Meyer's reset](http://meyerweb.com/eric/thoughts/2007/05/01/reset-reloaded/)
```

Comment:
Just a comment appart, but you should prefer to use [normalize.css](https://www.npmjs.com/package/normalize.css) rether than the Eric's reset.